README.md

```diff
@@ -1,79 +1,97 @@
 # Overview
-[LINE API Use Caseサイト](https://lineapiusecase.com/ja/top.html)で提供している[ヘアサロン予約](https://lineapiusecase.com/ja/usecase/reservation.html)のデモアプリケーションソースコードとなります。    
-今回紹介している手順を参考にすると、LINE APIを活用したヘアサロン予約アプリケーションを開発することが可能です。    
-ヘアサロン予約アプリケーションはLINEアプリでLIFFブラウザ上で起動し、ヘアサロンの予約を行うことが出来ます。  
-ユーザーの来店忘れを防止するための、LINEメッセージによるリマインド機能もデフォルトで実装できます。  
 
-なお、このページで紹介しているソースコードの環境はAWSを利用しています。  
-※ ドキュメントなどの文言は日本語対応となっています。  
-※ This document is written in only Japanese for now. We’ll translate it later as soon as possible.
+This is the demo app source code for the [Hair Salon Reservation](https://lineapiusecase.com/en/usecase/reservation.html) provided on the [LINE API Use Case site](https://lineapiusecase.com/en/top.html). By referring to the steps described in this article, you can develop a hair salon reservation app using the LINE API. The hair salon reservation app is launched on the LIFF browser of the LINE app, and you can make a reservation at the hair salon. To prevent users from forgetting to visit the store, you can implement a reminder function via LINE message by default.
```

Comment:
[nitpicking]
It sounds a bit like LIFF browser is a feature within the LINE app, so how about:
```suggestion
This is the demo app source code for the [Hair Salon Reservation](https://lineapiusecase.com/en/usecase/reservation.html) provided on the [LINE API Use Case site](https://lineapiusecase.com/en/top.html). By referring to the steps described in this article, you can develop a hair salon reservation app using the LINE API. The hair salon reservation app is launched on the LIFF browser in the LINE app, and you can make a reservation at the hair salon. To prevent users from forgetting to visit the store, you can implement a reminder function via LINE message by default.
```